### PR TITLE
[ACT-817] Add new preset type

### PR DESCRIPTION
**Pull request by valerieernst**
This PR adds the concept of categorizing presets as either automatic (i.e. automatically add when a user creates a destination) or associated with a specific pre-defined event.

See the README updates for more info.

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

- [ ] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [ ] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2189,6 +2189,20 @@
...
   integrity sha512-SId7GOdDFmm/B9ajIQpXELHW4OTbVvmJbOsoJkQOcUEtoZIiX2UWfk1v4BpKql8wJW9oAhzhIIru2Pv2Yxcg+w==
 
+"@segment/action-emitters@^1.0.1":
+  version "1.0.1"
+  resolved "https://registry.yarnpkg.com/@segment/action-emitters/-/action-emitters-1.0.1.tgz#8138513f32c1dff884413587ef671ff5160005d8"
+  integrity sha512-hEqafmqVuMlpV/GFt9r8h3VyUkbO00+3a6gCC/2ADwDCFWXfj28r9uo4cbNSoyQAp+U+70N19U37jMV2WKmq9A==
+  dependencies:
+    "@types/node" "^18.11.15"
+
+"@segment/analytics-browser-actions-sabil@^1.6.0":
```
Comment (SyedWasiHaider) [inline yarn.lock:2199]: This is weird..

File: packages/cli/src/commands/validate.ts [hunk 2]
```diff
@@ -105,10 +105,12 @@ export default class Validate extends Command {
       const actionFields = Object.keys(destination.actions[preset.partnerAction].fields ?? {})
 
       // Validate the FQL
-      const fqlError = this.validateFQL(preset.subscribe)
-      if (fqlError) {
-        this.isInvalid = true
-        errors.push(new Error(`The preset "${preset.name}" has an invalid \`subscribe\` query: ${fqlError.message}`))
+      if (preset.type === 'automatic') {
```
Comment (prayansh) [inline packages/cli/src/commands/validate.ts:108]: does `specificEvent` preset not have FQL to evaluate?

**Review by prayansh — APPROVED** [review]

File: packages/cli/src/commands/validate.ts (hunk 2 above)
Comment (SyedWasiHaider) [reply packages/cli/src/commands/validate.ts:108]: It doesn't at this point. We'll be generating the FQL based on the some inputs (like audience id) that we get later in the setup flow.

**Comment by joe-ayoub-segment** [comment]
Hi @valerieernst , 
I'm preparing the deploy for today - figuring out which Destinations need a Push. 
This PR changes all presets for all Destinations from what I can see (or a lot of them anyway). 
Does this mean that all these Destinations need to be Pushed?

**Pull request merged**